CONTRIBUTING.md

```diff
@@ -1,17 +1,28 @@
 # Getting Started
-If you are unsure about a possible contribution to the project contact the project owners about 
-your idea.
 
-Please see the [code of conduct](https://github.com/panoptes/POCS/blob/develop/CODE_OF_CONDUCT.md) 
-for our
-playground rules and follow them during all your contributions.
+We prefer that all changes to POCS have an associated
+[GitHub Issue in the project](https://github.com/panoptes/POCS/issues)
+that explains why it is needed. This allows us to debate the best
+approach to address the issue before folks spend a lot of time
+writing code. If you are unsure about a possible contribution to
+the project, please contact the project owners about your idea;
+of course, an [issue](https://github.com/panoptes/POCS/issues) is a
+good way to do this.
 
+Please see the
```

Comment:
Let's move this line to the top. Maybe even above the "Getting Started"

Reply:
Done